33-room-occupancy-monitor/README.md

```diff
@@ -0,0 +1,115 @@
+# Room Occupancy Monitor
+
+Receive notifications when motion is detected in a room and when the room's door is opened or closed.
+
+- [Room Occupancy Monitor](#room-occupancy-monitor)
+  - [You Will Need](#you-will-need)
+  - [Notehub Setup](#notehub-setup)
+  - [Sparrow Setup](#sparrow-setup)
+    - [Quickstart](#quickstart)
+    - [Hardware](#hardware)
+    - [Firmware](#firmware)
+  - [Testing](#testing)
+  - [Blues Community](#blues-community)
+  - [Additional Resources](#additional-resources)
+
+## You Will Need
+
+* [Sparrow Development Kit](https://shop.blues.io/products/sparrow-dev-kit)
+* 2 USB A to micro USB cables
+* [Magnetic Door Switch Set](https://www.sparkfun.com/products/13247)
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
+
+## Sparrow Setup
+
+### Quickstart
+
+Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) to get your Sparrow reference node paired with the gateway and associated with the Notehub project you just created. Note also that we'll only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the `hub.set` command in the quickstart.
+
+After you've completed the quickstart, leave the Notecarrier and essentials board powered and connected. These two devices will act as our gateway to Notehub, and we won't need to touch them again. The rest of this guide will focus on the reference node.
+
+### Hardware
+
+There are two primary pieces of hardware: the magnetic door switch and the PIR motion sensor attached to the Sparrow reference node's circuit board.
+
+The magnetic door switch consists of two plastic terminals, one of which has a pair of wires coming out of it. When the terminals are brought into contact (or close proximity), an internal [reed switch](https://en.wikipedia.org/wiki/Reed_switch) is closed and the two wires are electrically connected. As shown in the SparkFun product link above, the terminal with the wires is typically mounted to a door frame and the other terminal is mounted to the door such that when the door is closed, the two terminals are adjacent, closing the switch. Since we're just testing things out, don't mount the terminals yet. Instead, keep them on your work surface and plug the wires into the A1 and GND ports of the Sparrow reference node (it doesn't matter which wire goes into which port). When the switch is closed, A1 will connect to GND. Otherwise, A1 will be driven high by an internal pull-up.
+
```

Comment:
Just thinking of power use, wouldn't it be better to pull the signal down to ground and have it pulled high when the door is opened using the `normally-open` terminal?

Reply:
Doesn't it depend on which state the door is in more often: open or closed? I'm not sure what you mean by normally-open terminal.

Reply:
Sorry for the noise, I hadn't looked at the door switch you were using. I used this one, which has NO and NC connections so you can invert the polarity as desired. In my case the door was typically closed, so I used the NO contact so that the circuit is made when the door is open. 

https://www.amazon.com/Magnetic-Switch-Normally-Closed-Security/dp/B0735BP1K4/